src/libaktualizr/primary/initializer.cc

```diff
@@ -252,9 +252,9 @@ Initializer::Initializer(
       return;
     }
 
-    for (auto it = secondary_info_.begin(); it != secondary_info_.end(); ++it) {
-      it->second->Initialize();
-    }
+    //    for (auto it = secondary_info_.begin(); it != secondary_info_.end(); ++it) {
```

Comment:
Should these be removed instead of commented-out?

Reply:
Yeah.